File: platforms/iPad/Accounter iOS 1.0.2/Accounter/Result.m
//
//  Result.m
//  Accounter iOS2
//
//  Created by Amrit Mishra on 11/11/11.
//  Copyright 2011 __MyCompanyName__. All rights reserved.
//

#import "Result.h"


@implementation Result


-(id)init{
    
    
    self = [super init];
    if (self) {
        resultParts = [[NSMutableArray alloc]init];
        title = [[NSString alloc]init];
        hideCancel = false;
        showBack = false;
        
    }
    return self;
}

- (void)setResultParts:(NSMutableArray *)resultPartsList{
    resultParts = [[NSMutableArray alloc]initWithArray:resultPartsList];
}

- (NSMutableArray *)getResultParts{
    return resultParts;
}


- (void)setCookie:(NSString *)newCookie{
    
    cookie = [[NSString alloc]initWithString:newCookie];
}
- (NSString *)getCookie{
    return cookie;
}



-(void)setInputType:(int)newInputType{
    inputType = newInputType;
}
-(int)getInputType{
    return inputType;
}

-(void)setHideCancel:(BOOL)value{
    hideCancel = value;
}
-(BOOL)hideCancelValue{
    return hideCancel;
}

-(void)setShowBack:(BOOL)value{
    showBack = value;
}
-(BOOL)showBackValue{
    return showBack;
}


- (void)setTitle:(NSString *)newTitle{
    title = newTitle;
}
- (NSString *)getTitle{
    return title;
}

@end

File: platforms/blackberry/Accounter/src/com/vimukti/accounterbb/result/Result.java
package com.vimukti.accounterbb.result;

import java.util.Vector;

public class Result {
	private String title;
	private boolean showBack;
	private boolean hideCancel;
	
	private Vector resultParts = new Vector();
	private String cookie;
	
	public void setResultParts(Vector resultParts) {
		this.resultParts = resultParts;
	}

	public Vector getResultParts() {
		return resultParts;
	}

	public void setCookie(String cookie) {
		this.cookie = cookie;
	}

	public String getCookie() {
		return cookie;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getTitle() {
		return title;
	}

	public void setHideCancel(boolean hideCancel) {
		this.hideCancel = hideCancel;
	}

	public boolean isHideCancel() {
		return hideCancel;
	}

	public void setShowBack(boolean showBack) {
		this.showBack = showBack;
	}

	public boolean isShowBack() {
		return showBack;
	}


}
